docs/qix/README.md

```diff
@@ -57,27 +57,49 @@ enigma.connect(config).then((qix) => {
 enigma.js uses camel casing (`getObject`) for method names, while Engine API uses Pascal casing (`GetObject`).
 
 Parameters can be passed either by name through an object:
-```javascript
+
+```js
```

Comment:
I have another branch where I changed more to `js` so it was just out of habit, I'll revert